dev-libs/openssl/openssl-1.0.2h.ebuild

```diff
@@ -12,23 +12,16 @@ HOMEPAGE="http://www.openssl.org/"
 SRC_URI="mirror://openssl/source/${MY_P}.tar.gz"
 
 LICENSE="openssl"
+# subslot set to 1.0.2g version as this is the first release without SSLv2
+# support and thus breaks nearly every openssl consumer (see bug #575548)
```

Comment:
Yeah, I was a little confused by that. They have that comment in upstream, but don't appear to specify a subslot.